api/admin.py

```diff
@@ -59,6 +59,11 @@ class ContractAdmin(admin.ModelAdmin):
     )
     list_per_page = 100
     ordering = ("-modified_at",)
+    list_filter = (
+        "start_date",  
+        "end_date",    
+        "user",        
```

Comment:
filtering for user will not be practicable.
We need `search_filed` for it. We want to search for `first_name, `last_name`, `mail` and `personell_number` of a user.